lib/screens/settings_screen.dart

```diff
@@ -190,6 +190,9 @@ class SettingsListState extends State<SettingsList> {
           var val = SettingsEditorType.fromPublicString(publicStr);
           Settings.instance.defaultEditor = val;
           Settings.instance.save();
+          // TODO: Some dialog like
```

Comment:
I think you may have made the comment in the in-correct place. This block is about changing the default Editor.